features/relateddata.feature

```diff
@@ -0,0 +1,76 @@
+  Scenario: GET /relateddata and checking for many results
+    Given there is a Search API that gives a successful response and returns 3 results
+    And get page data request to zebedee for "/employmentandlabourmarket/peopleinwork/bulletin1" returns a page of type "bulletin" with status 200
+    And get breadcrumb request to zebedee for "/employmentandlabourmarket/peopleinwork/bulletin1" returns breadcrumbs 
+    And the search controller is running
+    When I navigate to "/employmentandlabourmarket/peopleinwork/bulletin1/relateddata"
+    Then the page should have the following content
+      """
+      {
+        "#main h1": "All data related to Labour Market statistics: March 2024",
+        ".ons-pagination__position": "Page 1 of 1",
+        ".ons-document-list h2": "Test Bulletin 0",
+        ".ons-breadcrumb": "Home  Economy  Gross Domestic Product (GDP)  Labour Market statistics"
+      }
+      """
+
+  Scenario: GET /relateddata  and breadcrumb request errors
+  Given there is a Search API that gives a successful response and returns 3 results
+  And get page data request to zebedee for "/employmentandlabourmarket/peopleinwork/bulletin1" returns a page of type "bulletin" with status 200
+  And get breadcrumb request to zebedee for "/employmentandlabourmarket/peopleinwork/bulletin1" fails
+  And the search controller is running
+  When I navigate to "/employmentandlabourmarket/peopleinwork/bulletin1/relateddata"
+  Then the page should have the following content
+    """
+    {
+      "#main h1": "All data related to Labour Market statistics: March 2024",
+      ".ons-pagination__position": "Page 1 of 1",
+      ".ons-document-list h2": "Test Bulletin 0",
+      ".ons-breadcrumb": "Labour Market statistics "
+    }
+    """
+
+  Scenario: GET /relateddata and it is not a page of type article, bulletin, or compendium_landing_page
+    Given get page data request to zebedee for "/economy/latest" returns a page of type "taxonomy_landing_page" with status 200
+    And the search controller is running
+    When I navigate to "/economy/relateddata"
+    Then the page should have the following content
+    """
+        {
+            "#main h1": "Page not found"
+        }
+    """
+
+  Scenario: GET /relateddata and has no related data
+    Given there is a Search API that gives a successful response and returns 0 results
+    Given get page data request to zebedee for "/employmentandlabourmarket/peopleinwork/bulletin2" does not have related data
+    And get breadcrumb request to zebedee for "/employmentandlabourmarket/peopleinwork/bulletin2" returns breadcrumbs 
+    And the search controller is running
+    When I navigate to "/employmentandlabourmarket/peopleinwork/bulletin2/relateddata"
+    Then the page should have the following content
+    """
+        {
+            "#main h1": "All data related to Labour Market statistics: March 2024",
+            "#no-results-text": "Sorry, there are no matching results."
+        }
+    """
```

Comment:
Could we add a check for no pagination here?

Reply:
👍